model/flow/transaction.go

```diff
@@ -53,14 +53,83 @@ func NewTransactionBody() *TransactionBody {
 }
 
 func (tb TransactionBody) Fingerprint() []byte {
+	authorizers := make([][]byte, len(tb.Authorizers))
+	for i, auth := range tb.Authorizers {
+		authorizers[i] = auth.Bytes()
+	}
+
+	// this comment below is to enforce that this struct
+	// definition is exactly sync'd with any others.  The
+	// constraint is enforced with a test.
+	///%[[ EQUALITY-PERF-CONSTRAINT {transaction-body-payload}
+	var payload = struct {
```

Comment:
since we're now reusing this struct in multiple places, could we just not use it as an anonymous struct, and just use it as a private struct in this file? Does that still give us the same gains?

Reply:
done